en/docs/apis/authenticating-and-authorizing-rest-apis.md

```diff
@@ -0,0 +1,86 @@
+# Authenticating and Authorizing REST APIs
+
+This section guides you through securing REST services and how requests
+to REST APIs are authenticated and authorized in WSO2 Identity Server.
+
+The requests that are sent via REST APIs are intercepted by tomcat
+valves and authenticated and authorized by an OSGI service. There are
+two OSGi services that provide the authentication and authorization
+service based on its own handlers.
+
+-   WSO2 Identity Server supports three ways of API authentication. 
+    -   Basic authentication: Uses the user’s credentials in the API invocation
+    -   OAuth 2 common flows: Obtains a token using an oauth2 flow and uses it to invoke the API
+    -   Client certificate-based: Uses Mutual SSL to authenticate in order to consume the APIs
+
+!!! note 
+    Unless one of the above authentication elements is sent in an API invocation request, the 401 Unauthorized HTTP response will be returned.
+    
+-   Authorization for the APIs is enforced at the endpoint level using **permissions**. Each secured endpoint has a predefined minimum level of permission that is required to be able to consume the endpoint. In order to access a particular endpoint, the user has to belong to a **role** that is in or above the defined permission level.
+
+!!! note
+    You can write your own handlers for both authentication and authorization and register them in OSGI.
+    
```

Comment:
shall we use separators between subheadings like we do in other docs?

Reply:
+1